### PR TITLE
37 component plus button in tabbar

### PR DESCRIPTION
**Pull request by peelerzin**
Component plus button ready. Also included in storybook. Styling was chose depending the color palette as they were no styling in figma for the plus button.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,58 +2,45 @@
...
-      "autoprefixer": "^10.4.13",
-      "babel-loader": "^8.3.0",
-      "postcss": "^8.4.21",
-      "prisma": "^4.10.1",
-      "tailwindcss": "^3.2.6",
-      "ts-node": "^10.9.1",
-      "ts-node-dev": "^2.0.0",
-      "typescript": "^4.9.5"
-    }
+  "resolutions": {
```
Comment (NoHara42) [inline package.json:5]: the up-to-date `package.json` is missing the prisma seed script

Comment (NoHara42) [reply package.json:5]: and other dependencies, can you reverse merge from main?

File: src/components/PlusButton.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+type PlusButton = {
+  add: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+export function PlusButton(props: PlusButton) {
+  return (
+    <div>
```
Comment (NoHara42) [inline src/components/PlusButton.tsx:9]: unnecessary wrapping div here

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see comments @peelerzin

File: src/components/PlusButton.tsx (hunk 2 above)
Comment (peelerzin) [reply src/components/PlusButton.tsx:9]: removed.

File: package.json (hunk 1 above)
Comment (peelerzin) [reply package.json:5]: I git pull main branch and then get back to this branch to merge main but Storybook doesn't display the tailwind design for the button component and defaults examples/settings appears. Inside of vs code if i reuse the button component in another page, the tailwind class works and  button appears with the tailwind class that i made, how comes?

Comment (NoHara42) [reply package.json:8]: @peelerzin pull the latest storybook changes from the main branch and they should fix this issue.

File: src/components/PlusButton.tsx (hunk 2 above)
Comment (NoHara42) [reply src/components/PlusButton.tsx:9]: @peelerzin please don't mark comments as resolved, this should be up to the reviewer

Comment (NoHara42) [reply src/components/PlusButton.tsx:9]: @peelerzin also this hasn't been resolved

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see change and reverse merge

**Comment by NoHara42** [comment]
@peelerzin button is not really round, can we fix this?
image

File: src/pages/login/index.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { HeaderTitle } from "@/components/HeaderTitle";
 import Head from "next/head";
+import { MouseEvent } from "react";
```
Comment (NoHara42) [inline src/pages/login/index.tsx:2]: unused import

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see small change

File: src/pages/login/index.tsx (hunk 3 above)
Comment (peelerzin) [reply src/pages/login/index.tsx]: done and reversed main

**Review by NoHara42 — APPROVED** [review]
LGTM

**Pull request merged**